weed/s3api/iceberg/iceberg_credential_vending_test.go

```
@@ -0,0 +1,115 @@
+func (v *stubVendor) VendTableCredentials(_ context.Context, principal, bucket, prefix string) (*VendedCredentials, error) {
+	v.gotcaller, v.gotBucket, v.gotPrefix = principal, bucket, prefix
+	return v.credentials, v.err
+}
+
+func vendingServer(vendor CredentialVendor) *Server {
+	s := &Server{s3Endpoint: "http://s3.example:8333"}
+	s.SetCredentialVendor(vendor)
+	return s
+}
+
+func delegationRequest() *httptest.ResponseRecorder {
```

golangci-lint (2.12.2)

[error] func delegationRequest is unused

(unused)